config.json

```diff
@@ -1,42 +1,54 @@
 {
-    "web": {
-        "port": 3000,
-        "title": "BitCannon",
-        "torrentsPerPage": 100,
-        "defaultSearchSorting": "seeders",
-        "defaultSearchOrder": "desc"
-    },
-    "database": {
-        "mongodb": {
-            "enabled": true,
-            "host": "localhost",
-            "port": 27017,
-            "collection": "bitcannon"
-        }
+  "web": {
+    "port": 3000,
+    "title": "BitCannon",
+    "torrentsPerPage": 100,
+    "defaultSearchSorting": "seeders",
+    "defaultSearchOrder": "desc"
+  },
+  "database": {
+    "mongodb": {
+      "enabled": true,
+      "host": "localhost",
+      "port": 27017,
+      "collection": "bitcannon"
+    }
+  },
+  "torrents": {
+    "whitelist": {
+      "enabled": false,
+      "categories": []
     },
-    "session": {
-        "secret": "asdasdasdasd43jn5kj2n43k"
+    "blacklist": {
+      "enabled": false,
+      "categories": []
+    }
+  },
+  "session": {
+    "secret": ""
```

Comment:
Sounds like a good idea. I was also thinking maybe we should rebrand as a different app since this isn't really like the original and the original guy that made this isn't really involved. Thoughts?